### PR TITLE
FT20-388 BE - Create and endpoint for POST client address

### PR DESCRIPTION
**Pull request by joselsanabria**

### CODE REVIEW

File: src/server.js [hunk 1]
```diff
@@ -56,6 +56,24 @@ app.get("/clients/:clientId/products", (req, res) => {
   ]);
 });
 
+
+app.post("/clients/:clientId/address", (req, res) => {
+  const { firstName, lastName, emailAddress, country, streetAddress, city, state, zipCode } = req.body;
+  
+  if (!firstName || !lastName || !emailAddress || !country || !streetAddress || !city || !state || !zipCode) return res.sendStatus(400);
```
Comment (maacaro) [inline src/server.js:63]: @joselsanabria los bloques if siempre con llaves 
```
if(x>0){
 lo que sea
}
```

File: src/server.js [hunk 2]
```diff
@@ -56,6 +56,24 @@ app.get("/clients/:clientId/products", (req, res) => {
...
+
+  res.json({
```
Comment (maacaro) [inline src/server.js:65]: @esto no lleva un return?

**Review by maacaro — CHANGES_REQUESTED** [review]
@joselsanabria  muy bien los test por alli deje un par de comments de estylo en el codigo

File: src/server.js (hunk 1 above)
Comment (joselsanabria) [reply src/server.js:63]: if (!firstName || !lastName || !emailAddress || !country || !streetAddress || !city || !state || !zipCode) return res.sendStatus(400);{
    res.json({
      clientId: 1,
      firstName,
      lastName,
      emailAddress,
      country,
      streetAddress,
      city,
      state,
      zipCode,
    });
  }

File: src/server.js (hunk 2 above)
Comment (joselsanabria) [reply src/server.js:65]: if (!firstName || !lastName || !emailAddress || !country || !streetAddress || !city || !state || !zipCode) return res.sendStatus(400);{
    res.json({
      clientId: 1,
      firstName,
      lastName,
      emailAddress,
      country,
      streetAddress,
      city,
      state,
      zipCode,
    });
  }

**Review by maacaro — APPROVED** [review]
LGTM

**Pull request merged**